casa/OS/Memory.h

```diff
@@ -110,17 +110,20 @@ class Memory
     // How much memory has been allocated by the programmer (via either
     // <src>new</src> or <src>malloc</src>. This can include some extra
     // overhead bytes, e.g. for alignment reasons.
-    static size_t allocatedMemoryInBytes();
+    [[deprecated("Results are not accurate (see help for mallinfo2()), so avoid this call")]]
+    static size_t allocatedMemoryInBytes() { return 0; }
```

Comment:
Instead of returning 0, shouldn't we raise an exception upon use, or just remove the function?

Reply:
I thought that this value is probably only useful to report to the user.  If e.g. casa uses it for that, it's probably better to show '0' with a compilation warning, then to actually crash...